File: App.API/Models/RoomService.cs
using System;
using System.Collections.Generic;

namespace App.API.Models
{
    public class RoomService
    {
        public int Id { get; set; }
        public int BookingId { get; set; }
        public DateTime ServiceDate { get; set; }
        public bool IsCompleted { get; set; }
        public bool IsPaid { get; set; }
        public decimal TotalPriceBeforeTax { get; set; }
        public decimal TaxAmount { get; set; }
        public decimal TotalPrice { get; set; }
        public Booking Booking { get; set; }
        public ICollection<RoomServiceItem> RoomServiceItems { get; set; }
    }
}

File: App-SPA/src/app/_models/roomService.ts
import { RoomServiceItem } from './roomServiceItem';

export interface RoomService {
    id: number;
    bookingId: number;
    firstName?: string;
    lastName?: string;
    roomNumber?: string;
    serviceDate: Date;
    isCompleted: boolean;
    isPaid: boolean;
    totalPriceBeforeTax: number;
    taxAmount: number;
    totalPrice: number;
    roomServiceItems: RoomServiceItem[];
}
